[issue/18] - Added results and discussion content (#26)
teamulster2/report: report/05_results_and_discussion/00_result_and_discussion.md

# Results and discussion

> In this project, a new application has now been implemented that is 

Reviewer: ```suggestion
In this project, a new application has been implemented that is 
```

based on the results of the report from [4]. As described in the 
Methodology (3) chapter, a Flutter application is available, that is 
initially geared and adapted to an Android application. This app can 
easily be expanded to an iOS or web application with the same code base. 
In addition, a server is available for evaluation purposes to which 
the data can be transmitted.

> Above all, the results of the pvt and spatial span task in combination 

Reviewer 2: ```suggestion
Above all, the results of the pvt and spatial-span-task in combination 
```

with the current mood and current activity should be able to provide 

> meaningful results to the people, who are responsible for the study. 

Reviewer: ```suggestion
meaningful results for upcoming studies using this application.
```

In addition, the storage of the data from the questionnaire has been 
optimized and the questions can be exchanged.

Interchangeability is particularly important for the final result 
of the report. The app is not only designed for a single study, so 

> the values ​​can be adjusted. In addition, the data exchange is kept 
> anonymous, so that the content can only be used for study purposes 
> and no conclusions can be drawn about actual people. Both the client 

Reviewer 2: ```suggestion
the values ​​can be adjusted. In addition, privacy was kept in mind while
designing the hole system, even though it is technically possible to track
people over there public IP, no mechanism has been implemented to
collect this or other research unrelated data. Both the client 
```

application and the server side were designed to be interchangeable 
and adaptable.

Possible further steps would now be to design an admin page so that 

> changes to the config of a study do not have to be made via a terminal 
> window, but at best can be made via a desktop application or a website. 

Reviewer: ```suggestion
changes to the study config can be done more comfortable by extending the command
line interface with a desktop application or website.
```

> The existing access via https can be expanded and adapted for 

Reviewer 3: ```suggestion
The existing access via http can be expanded and adapted for 
```

> this purpose.

Reviewer 3: Another improvement can be a audio recognition test. So future studies have more data to evaluate.

Author: I added this approach to the next chapter (conclusion).

Reviewer 3: ok perfect :+1:
